gef-ui/templates/partials/mandatory-criteria.njk

```diff
@@ -25,7 +25,7 @@
     Confirm eligiblity (mandatory criteria)
   </h1>
 
-  <div data-cy="mandatory-criteria">
+  <div data-cy="mandatory-criteria" class="app-prose-scope">
```

Comment:
what's the meaning of app-prose-scope ?  LGTM otherwise

Reply:
Stole it from the prototype website as p tags arent being styled. I guess we shouldnt be using it, therefore need to resolve this issue regarding the missing stylesheet